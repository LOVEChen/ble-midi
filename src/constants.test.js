/* eslint-env jest */

test('constants', () => {
  const constants = require('./constants');
  expect(constants).toMatchSnapshot();
});
