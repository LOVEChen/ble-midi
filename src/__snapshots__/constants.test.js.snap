// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants 1`] = `
Object {
  "DEFAULT_SERVICE_NAME": "skratchdot-ble-midi",
  "MIDI_IO_CHARACTERISTIC_UID": "7772E5DB-3868-4112-A1A9-F2669D106BF3",
  "MIDI_SERVICE_UID": "03B80E5A-EDE8-4B33-A751-6CE34EC4C700",
  "SYSEX_END": 247,
  "SYSEX_START": 240,
}
`;
